.eslintrc.cjs

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  env: {
```

Comment:
Do you need this?

Reply:
I had it for eslint-plugin-jest, not currently needed so I removed it :+1: